qa/test-old-engine/pom.xml

```diff
@@ -176,7 +176,7 @@
         </dependency>
 
         <dependency>
-          <groupId>org.codehaus.groovy</groupId>
+          <groupId>org.apache.groovy</groupId>
```

Comment:
💭 We are running the tests with process engine 7.21, I would expect here too to preserve Groovy 2 and update the dependency in next minor 7.23. Similar to migration fixture 7.10, keep `${version.groovy.old}`. Or should we run the tests with Groovy 4?

Reply:
It is a good point. 
I think it is fine to run the tests on the old engine using the new Groovy version. As the test are passing meaning they are independent of the Groovy version.

Reply:
👍